zhaquirks/sinope/sensor.py

```diff
@@ -45,16 +45,16 @@ class SinopeManufacturerCluster(CustomCluster):
 class SinopeTechnologiesIasZoneCluster(CustomCluster, IasZone):
     """SinopeTechnologiesIasZoneCluster custom cluster."""
 
-    class ZoneStatus(t.enum8):
-        """zone_status values."""
+    class LeakStatus(t.enum8):
+        """leak_status values."""
 
         Dry = 0x00
         Leak = 0x01
 
     attributes = IasZone.attributes.copy()
     attributes.update(
         {
-            0x0030: ("zone_status", ZoneStatus, True),
+            0x0030: ("leak_status", LeakStatus, True),
```

Comment:
Haven't checked ZHA yet, but did this quirk rely on HA creating a binary sensor for an attribute called `zone_status`? (And possibly the higher attribute id overwriting the actual `zone_status`?)

So, is another "binary sensor entity" necessary in ZHA for this `leak_status` now?

Reply:
You're right, it does rely on an IASZone `zone_status` attribute existing. I'll add a new matcher should to ZHA in the corresponding PR.